psi4/share/psi4/fsapt/fsaptdiff.py

```diff
@@ -57,62 +57,62 @@
 
     fh = open('%s/fsapt.dat' % dirA, 'w')
     fh, sys.stdout = sys.stdout, fh
-    print '  ==> F-ISAPT: Links by Charge <==\n'
+    print('  ==> F-ISAPT: Links by Charge <==\n')
     stuffA = computeFsapt(dirA, False)
-    print '   => Full Analysis <=\n'
+    print('   => Full Analysis <=\n')
     printOrder2(stuffA['order2'], stuffA['fragkeys']) 
-    print '   => Reduced Analysis <=\n'
+    print('   => Reduced Analysis <=\n')
     printOrder2(stuffA['order2r'], stuffA['fragkeysr']) 
```

Comment:
`print "Hello,", "world!"` doesn't work :). As the forums showed we are no long Python 2.6 compatible, which is fine. Do we require 2.7 in CMake?